### ISSUE TITLE
JOB NA ENTREGAFARMA

### ISSUE THREAD

**Issue by seujobentrega** [seujob/hot#160]
[Empresa](https://api.github.com/repos/react-brasil/vagas/issues/794) [Challenge](https://api.github.com/repos/react-brasil/vagas/issues/794) [Contato](https://api.github.com/repos/react-brasil/vagas/issues/794) *[Anuncie](https://seujob.github.io/hot)* ![.](https://raw.githubusercontent.com/seujobentrega/media/master/9e6ec07820bbbd6a1546a5e6350f02d65d5d817f)

**Comment by angelorubin** [comment]
E como me candidato a vaga?

**Comment by enieber** [comment]
@angelorubin acho que essa vaga já foi fechada

**Comment by seujob** [comment]
@angelorubin estou implementando um algoritmo para atribuir o link de inscrição ao botão Challenge! (contudo links que carregam imagens podem quebrar as publicações, esta solução ainda está sendo encaminhada)

**Comment by seujob** [comment]
@enieber foi fechada sim! outra solução under development é fechar issues e não apresentar closed issues no feed, contudo estou estudando uma solução mais ubíqua

**Comment by seujob** [comment]
@angelorubin outro detalhe importante é que os links nessa publicação estão apontando para api do github mas isso foi corrigido nas publicações mais recentes!

**Comment by angelorubin** [comment]
Quando eu clico em algum dos links eles levam a uma pagina do github com um json..

**Comment by seujob** [comment]
@angelorubin exatamente, esse é o retorno da api do github, foi um bug no caso dessa issue mas tentei resolver nas recentes =)

**Comment by seujobentrega** [comment]
@enieber @angelorubin podem verificar agora se aparece alguma badge na imagem da issue indicando que ela foi fechada?

**Comment by enieber** [comment]
Ja sim. Ta ficando massa o projeto parabens

**Comment by seujob** [comment]
@enieber muito obrigado meu irmão, estou apreciando esta experiência! :) vou implementar um link para o criador do job poder fechar a issue e gerar essa badge =) também verifiquei que atualizou no slack mas apenas quando ele limpa o cache
